### PR TITLE
Update AbstractMessage.java

### PR DESCRIPTION
**Pull request by Kentzo**
Method Builder#clone() has been changed to bypass Java 1.6 compiler issue.

google/protobuf#2428

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by Kentzo** [comment]
@goorov You need to sign https://cla.developers.google.com/clas with your email.

**Comment by Kentzo** [comment]
I signed it!

**Comment by goorov** [comment]
I signed it!

File: java/core/src/main/java/com/google/protobuf/AbstractMessage.java [hunk 1]
```diff
@@ -328,8 +328,11 @@ UninitializedMessageException newUninitializedMessageException() {
       extends AbstractMessageLite.Builder
       implements Message.Builder {
     // The compiler produces an error if this is not declared explicitly.
+    /** Method isn't abstact to bypass Java 1.6 compiler issue. */
```
Comment (xfxyjwf) [inline java/core/src/main/java/com/google/protobuf/AbstractMessage.java]: Use "// " style comment to be consistent with the line above. Also put a link to the Java issue here?

**Comment by xfxyjwf** [comment]
@goorov Can you help send a pull request instead? That would make it a little easier for the CLA bot.

**Comment by goorov** [comment]
As aт initial commit author I confirm it's OK to submit this pull request by @Kentzo.

**Comment by Kentzo** [comment]
@xfxyjwf Closeing PR, @goorov will re-make it on his own.

**Comment by goorov** [comment]
@xfxyjwf The new PR with same changes has been created https://github.com/google/protobuf/pull/2602

**Pull request closed without merging**